common/searchattribute/defs.go

```diff
@@ -81,6 +81,14 @@ const (
 	// define a custom ScheduleId search attribute, in which case the query using the ScheduleId would operate just like
 	// any other custom search attribute.
 	ScheduleID = "ScheduleId"
+
+	// PausedEntities is a search attribute that stores the list paused entities in the workflow.
+	// Format of a single paused entity: "<entity_type>:<entity_id>:<policy_id>".
+	//  * entity_type can be activity, workflow, etc.
+	//  * entity_id is based on type. For activity it will be activity type, for workflow it will be workflow id.
+	//  * policy_id is an arbitrary string.
+	// example for paused activities, manual pause: "activity:MyCoolActivityType:MANUAL"
+	PausedEntities = "PausedEntities"
```

Comment:
Need to figure how to express the new convention of `system.` search attributes.

Reply:
The `system.` prefix is gonna be a separate work, and it should work seamlessly with the existing search attributes as well. I think it will be part of the query converter when parsing the query. I don't think we need to worry about it in this PR.

Reply:
we agree on adding "Temporal" prefix to the search attribute.